src/index.js

```diff
@@ -28,22 +34,43 @@ form.addEventListener('submit', (e) => {
   if (!desc) return;
   taskList.appendChild(createTaskDOM(desc));
   form.children[0].value = '';
+  const newTask = new Task(desc, toDoList.taskList.length);
+  toDoList.addTask(newTask);
+  storage.save('tasks', toDoList.taskList);
 });
 
 const checkboxes = document.querySelectorAll('.checkbox');
 checkboxes.forEach((box) => {
   box.addEventListener('change', () => {
     const boxIndex = Array.prototype.indexOf.call(checkboxes, box);
-    toDoList.taskList[boxIndex].toggleCompleted();
+    toDoList.taskList[boxIndex].completed = !toDoList.taskList[boxIndex].completed;
   });
 });
 
```

Comment:
- [ ] When I check a checkbox it should change the `completed` value to `true` in the `localStorage` and back to `false` when I uncheck it but it does not happen. Kindly fix that.